pkg/porter/install_test.go

```diff
@@ -55,9 +57,11 @@ func TestPorter_applyDefaultOptions_DebugOff(t *testing.T) {
 	require.NoError(t, err)
 
 	opts := &InstallOptions{
-		sharedOptions: sharedOptions{
-			bundleFileOptions: bundleFileOptions{
-				File: "porter.yaml",
+		BundleLifecycleOpts{
```

Comment:
Just a suggestion, in some places I've switched to doing this because the deeply nested struct inits are getting hard to for me to manage

```go
opts := &InstallOptions{}
opts.File = "porter.yaml"
```